modules/nw-openshift-sdn-modes.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/about-openshift-sdn.adoc
+
+[id="nw-openshift-sdn-modes_{context}"]
+= OpenShift SDN network isolation modes
+
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+OpenShift SDN provides three SDN modes for configuring the pod network:
```

Comment:
See comment above (to "the" or not to "the"?)

Reply:
@bobfuru, I think this is _OpenShift SDN_ vs _the OpenShift SDN network provider_. The latter seems odd as just _OpenShift SDN network provider_, I suppose. Same with _OVN-Kubernetes_ vs _the OVN-Kubernetes_.

Reply:
Oh, yes...that makes sense. Thanks!